scripts/common-apps-functions-source.sh

```diff
@@ -318,14 +316,15 @@ function build_llvm()
 
   mkdir -pv "${LOGS_FOLDER_PATH}/${llvm_folder_name}"
 
+  local llvm_patch_file_name="llvm-${ACTUAL_LLVM_VERSION}.patch.diff"
```

Comment:
You are reverting a previous commit, I would say you are not using the very latest version of the repo.

Reply:
Yep, should be fixed now.